### ISSUE TITLE
About WSI reconstruction and registration

### ISSUE THREAD

**Issue by Luffy03** [SunnyHaze/miSRGAN-pytorch#2]
Hello, many thanks for your great work!
It seems that this repo only re-produces the reconstruction of MRI, is it right? Can the model be used to WSI volume reconstruction and MRI-WSI registration as the original paper?

**Comment by SunnyHaze** [comment]
Hi, thanks for your attention.
Since the high-quality dataset with multi-direction information for registration has not been publicly released, it's hard for this repo to perform experiments.

Thus, only reconstruction experiments are reproduced.

**Comment by Luffy03** [comment]
Many thanks for your quick reply!
About reconstruction, can this model apply to WSI?

**Comment by SunnyHaze** [comment]
Indeed, I am not an expert in the medical field, so I haven't had hands-on experience. However, as you can see, this method theoretically applies to any task that requires super-resolution interpolation in a particular coordinate direction. All I can say is that it is worth trying, and if you achieve good results, you can also provide feedback to the Research Society via the issue tracker. Thank you.

Good luck!

**Comment by Luffy03** [comment]
Thank you very much!
